src/Console/WarningsDetector.php

```diff
@@ -57,13 +57,6 @@ public function detectOldVendor()
         }
     }
 
-    public function detectXdebug()
-    {
-        if (extension_loaded('xdebug')) {
-            $this->warnings[] = 'You are running PHP CS Fixer with xdebug enabled. This has a major impact on runtime performance.';
```

Comment:
I would still expect this if `PHP_CS_FIXER_ALLOW_XDEBUG` is set to be honest...

Reply:
Because using `PHP_CS_FIXER_ALLOW_XDEBUG` means you enable Xdebug intentionally. I would at least add another environment variable that allows to disable the message.